.github/workflows/ci.yml

```diff
@@ -21,7 +21,6 @@ jobs:
       fail-fast: false
       matrix:
         java-version:
-          - 11
```

Comment:
We should keep 11 until next LTS is out.

Reply:
We can discuss more offline, but JDK 15 is an [Azul MTS release](https://www.azul.com/products/azul-support-roadmap/):

> For those releases designed as MTS, Azul provides Production Support for at least 1.5 years after the general availability of the next LTS release, plus an additional 1 year of Extended Support.

These MTS releases are fully open source and available to everyone. They did this for JDK 13 as well. Assuming this pattern holds for the future, it will allow us to update every year.

Note that LTS for JDK 11 is an Oracle designation, for their commercial releases. The OpenJDK project itself does not provide LTS releases or any builds outside of the six month release window. These come from the various OpenJDK vendors who contribute to the update branches like `15u`.